src/main/java/com/google/sps/data/User.java

```diff
@@ -19,14 +18,10 @@ public User(MapLocation geolocation) {
   * @return User with zipCode
   **/
   public User(int zipCode) {
-    this.zipCode = zipCode;
+    // TODO: Convert zipcode to MapLocation
```

Comment:
I think you can keep the assignment from the existing version, and create a private method to handle the conversion.

Reply:
I abstracted out the helper function, but I think that this conversion may have to take place in the javascript anyways in order to use the Geocoding API to convert from zipcode to coordinates.